docs/src/_cloud/cloud-cli.md

```diff
@@ -31,6 +30,38 @@ Logging out of Meltano Cloud invalidates your login token, and deletes the local
 meltano-cloud logout
 ```
 
+## `config`
+
+The `config` command provides an interface for managing project configuration and secrets.
+Config supports the setting of Environment Variables via the `env` command group.
+
+### `env`
+
+Values set via this interface will be injected as Environment Variables into tasks run within the associated project.
+Once `set`, values cannot be viewed.
+If you are unsure of the current value of an env var, use the `set` command to set a known value.
```

Comment:
Do you intend for these lines to be rendered with line breaks between them? If so, you'll need to add a blank line between each. If not, these should be combined into a single line.

Reply:
We don't exactly have a markdown style guide yet, but past feedback has been that one sentence per line is preferable as i) it is rendered the same as continuous lines (i.e. in html paragraphs) but ii) it makes PRs much cleaner and easier to review (fewer awkward mid-line changes on really long lines). As markdown is a code artefact managed in code editors, rather than WYSIWYG, I'd be inclined to advocate for one-sentence-per-line as a standard rather than reverting 🤔